package.json

```diff
@@ -13,9 +13,9 @@
     "@types/node": "^16.11.1",
     "@types/react": "^17.0.30",
     "@types/react-dom": "^17.0.9",
-    "bootstrap": "^5.1.1",
+    "bootstrap": "^5.2.3",
     "react": "^17.0.2",
-    "react-bootstrap": "^1.6.4",
+    "react-bootstrap": "^2.7.0",
```

Comment:
I think these package changes should be separated out into its own pr so there’s less of a chance for a Bootstrap package issue to be lead back to your Contact page work, @colinwilliams91.